### PR TITLE
Fix parameters defaults & deploy code

### PR DESCRIPTION
**Pull request by karimodm**
# Description of change

Change a bunch  of deploy stuff and defaults to launch OTV in good shape.

### CODE REVIEW

File: plugins/pow/parameters.go [hunk 1]
```diff
@@ -9,7 +9,7 @@ import (
 // ParametersDefinition contains the definition of the parameters used by the pow plugin.
 type ParametersDefinition struct {
 	// Difficulty defines the PoW difficulty.
-	Difficulty int `default:"22" usage:"PoW difficulty"`
```
Comment (jonastheis) [inline plugins/pow/parameters.go:12]: This should also be changed in `config.default.json` and possibly across documentation

File: plugins/messagelayer/parameters.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ type ParametersDefinition struct {
 	}
 
 	// TangleTimeWindow defines the time window in which the node considers itself as synced according to TangleTime.
-	TangleTimeWindow time.Duration `default:"2m" usage:"the time window in which the node considers itself as synced according to TangleTime"`
```
Comment (jonastheis) [inline plugins/messagelayer/parameters.go:23]: This should only be changed for our deployment. In general I think 2min is okay

File: plugins/pow/parameters.go (hunk 1 above)
Comment (karimodm) [reply plugins/pow/parameters.go:12]: Done, I couldn't find references to this number in the docs.

File: plugins/messagelayer/parameters.go (hunk 2 above)
Comment (karimodm) [reply plugins/messagelayer/parameters.go]: Done

**Review by jonastheis — APPROVED** [review]

**Review by jonastheis — APPROVED** [review]

**Pull request merged**